src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -141,6 +140,16 @@ export default class QuerySummary extends React.Component<{ routingStore:Extende
         getGAInstance()('send', 'event', 'resultsView', 'query modified');
     }
 
+    @computed get queryForm(){
+        return <div style={{marginTop:10}}>
```

Comment:
welcome to the dark side 😈

Reply:
why is this the dark side?  jsx from @computed?  if so, i've long been guilty.  the darkside is jsx from mobxpromise.